source/elements/oneDPL/source/parallel_api/iterators.rst

```diff
@@ -174,6 +221,13 @@ to index into the index map. The corresponding value in the map is then used
 to index into the value set defined by the source iterator. The resulting lvalue is returned
 as the result of the operator.
 
+``permutation_iterator`` is SYCL device-copyable if both the ``SourceIterator`` and the ``IndexMap``
+are SYCL device-copyable. ``permutation_iterator`` is *indirectly device accessible* if both the
+``SourceIterator`` and the ``IndexMap`` are *indirectly device accessible*.
+
+When using ``permutation_iterator`` in combination with an algorithm with a ``device_policy``, ``SourceIterator`` must
+be *indirectly device accessible*.
```

Comment:
For our other iterator adaptors, we state that the source iterator must satisfy `AdaptingIteratorSource`. Should we say the same here for `SourceIterator`?

Reply:
We do say that, but it is a few paragraphs above.  It is odd though that there is so much separation between requirements of the same component iterator. I'll try to group them together.

Reply:
I missed it originally. The grouping is a good improvement.